### PR TITLE
[ENG-1119] - Open folder shortcut + select item with arrow down

### PR DESCRIPTION
**Pull request by ameer2468**
**This PR does the following**:

- Meta key + Arrow up keys execute double clicking on any folder or file to open.
- To support the shortcut above, I also added the ability to press Arrow Down on your keyboard, to select the first item in Explorer **and you can continue using the keyboard**.

**This means now, when you navigate folders, you can continue to navigate using just your keyboard.**

### CODE REVIEW

File: interface/app/$libraryId/Explorer/View/GridList.tsx [hunk 1]
```diff
@@ -248,6 +248,14 @@ export default ({ children }: { children: RenderItem }) => {
...
 		}
 
+		if (e.key === 'ArrowDown' && explorer.selectedItems.size === 0) {
+			const item = grid.getItem(0);
+			if (!item?.data) return;
+			explorer.resetSelectedItems([item.data]);
+			activeItem.current = item.data;
+			return;
+		}
+
```
Comment (niikeec) [inline interface/app/$libraryId/Explorer/View/GridList.tsx:258]: needs to update selecto targets

File: interface/app/$libraryId/Explorer/View/ListView.tsx [hunk 2]
```diff
@@ -819,6 +819,12 @@ export default () => {
 
 		const range = getRangeByIndex(ranges.length - 1);
 
+		if (e.key === 'ArrowDown' && explorer.selectedItems.size === 0) {
+			const item = rows[0]?.original;
+			if (item) explorer.addSelectedItem(item);
+			return;
+		}
+
```
Comment (niikeec) [inline interface/app/$libraryId/Explorer/View/ListView.tsx:827]: need to update ranges

**Review by niikeec — CHANGES_REQUESTED** [review]

File: interface/app/$libraryId/Explorer/TopBarOptions.tsx [hunk 3]
```diff
@@ -26,23 +25,30 @@ export const useExplorerTopBarOptions = () => {
...
 	const os = useOperatingSystem();
-	const keybind = keybindForOs(os);
+	const controlSymbol = (letter: string) => {
+		return [
+			os === 'macOS'
+				? modifierSymbols[ModifierKeys.Meta][os]
+				: modifierSymbols[ModifierKeys.Control]['Other'],
+			letter
+		] as string[];
+	};
```
Comment (utkubakir) [inline interface/app/$libraryId/Explorer/TopBarOptions.tsx:35]: let's move this to util/keybind.ts with some description of what it does

**Review by utkubakir — APPROVED** [review]
lgtm

**Review by niikeec — APPROVED** [review]

**Pull request merged**